lms/djangoapps/student_account/test/test_views.py

```diff
@@ -19,11 +19,13 @@
 from django.http import HttpRequest
 from edx_rest_api_client import exceptions
 
-from course_modes.models import CourseMode
 from commerce.models import CommerceConfiguration
 from commerce.tests import TEST_API_URL, TEST_API_SIGNING_KEY, factories
 from commerce.tests.mocks import mock_get_orders
 from openedx.core.djangoapps.programs.tests.mixins import ProgramsApiConfigMixin
+from nose.plugins.attrib import attr
+
+from course_modes.models import CourseMode
 from openedx.core.djangoapps.user_api.accounts.api import activate_account, create_account
 from openedx.core.djangoapps.user_api.accounts import EMAIL_MAX_LENGTH
 from openedx.core.djangolib.js_utils import dump_js_escaped_json
```

Comment:
Nit: Import order is a little wonky here.  stdlib first, then external libraries, then stuff in edx-platform.  One blank line between each group. `nose` should come in the same group as `django`.  Each group should be alphabetical.